src/Tribe/REST/V1/Validator/Interface.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+
+interface Tribe__Events__REST__V1__Validator__Interface  extends Tribe__Events__Validator__Interface {
```

Comment:
Not immediately, no.
My idea here is to make REST V1 classes depend on a REST V1 specific interface to grant flexibility in the future.
While it currently could add empty code it will leave us free to specialize the validator API and methods (in terms of signature) down the road keeping the context all in the same place.